docs/reference/snapshot-restore/apis/get-snapshot-api.asciidoc

```diff
@@ -134,12 +134,17 @@ Sort order. Valid values are `asc` for ascending and `desc` for descending order
 (Optional, string)
 Offset identifier to start pagination from as returned by the `next` field in the response body.
 
+`offset`::
+(Optional, integer)
+Numeric offset to start pagination from. Using a non-zero value for this parameter is mutually exclusive with using the `after` parameter.
```

Comment:
Perhaps worth clarifying the offset is evaluated after (name) filtering?

Also, I thought we had a case of needing to use both `after` and `offset`, but if not, I am fine with the restriction.